php-frontend/src/test/java/org/sonar/php/metrics/CpdVisitorTest.java

```diff
@@ -214,6 +214,22 @@ public void should_not_restore_from_cache_when_string_table_are_null() throws IO
     assertThat(actual).isFalse();
   }
 
+  @Test
+  public void should_not_compute_cpt_tokens_for_attributes() throws IOException {
```

Comment:
We should apply the same naming convention we have for `sonar-iac` and also for `sonar-php`. So maybe you want to rename it to camelCase as well.